### ISSUE TITLE
Filter columns form GET query data

### ISSUE THREAD

**Issue by andreas-glaser** [APY/APYDataGridBundle#829]
As i'm using FOSRestBundle and have format agnostic controllers I'd like to apply the filters by GET instead of POST. Is that possible? (I can't find any configuration option for it)

**Comment by qferr** [comment]
I agree with you. The method POST should be uses for the mass action, export and tweaks. @APY/collaborators ?

**Comment by hmert** [comment]
I agree with you, plus one may want also cache `GET` requests for performance issues.

**Comment by andreas-glaser** [comment]
Great! One other aspect is of course that filters could be shared or bookmarked.

**Comment by qferr** [comment]
@APY/collaborators I'm working on this issue. 

When a grid is filtered, ordered, limited, the page is reset to zero ([see](https://github.com/APY/APYDataGridBundle/blob/master/Grid/Grid.php#L828)). Why ? 

All filters are removed from the URL. We could not shared or bookmarked a grid with a page, order or limit.

**Comment by volodyawork** [comment]
I use this bundle in my local project.
Filtering with GET parameters would be very useful.
Thanks for a bundle!

**Comment by felixfidesio** [comment]
@qferr Do you have an ETA on this?
